core/src/main/resources/swagger/rawls.yaml

```diff
@@ -4277,26 +4279,63 @@ definitions:
     required:
       - email
       - accessLevel
+      - canShare
     properties:
       email:
         type: string
         description: email address of the user or group whose permissions will be changed
       accessLevel:
         type: string
         description: The access level to grant to this user or group (OWNER, READER, WRITER, NO ACCESS)
+      canShare:
+        type: boolean
+        description: Set to true if you want this user to be able to share the workspace with other users
+
+  WorkspaceACLUpdateResponseList:
```

Comment:
this particular addition was forgotten in the firecloud invites PR. i did make one small change to this response- i differentiate invitesSent from invitesUpdated so orchestration doesn't potentially send out too many invites. small issue so i fixed it here. orch has been updated accordingly